ui/src/message_popup/popup_question.jsx

```diff
@@ -68,16 +72,14 @@ export default React.createClass({
   onDonePressed(text=undefined) {
```

Comment:
It might help clarify to flatten the nested ifs here, something like this:

```
if (!this.props.shouldShowSummary) {
  this.props.onDone(this.state.elapsedMs);
  return;
}

if (this.state.isDoneReviewing) {
  this.props.onDone(this.state.elapsedMs);
  return;
}

...
```